wiki/Welcome/en.md

```diff
@@ -1,148 +1,38 @@
-
-## Competitiveness
-This may look like a simple game of clicking circles and holding sliders, but it can get really intense and serious! Here's a few things you should know about osu! rank system.
-
-### Levels
-The player's level only represents the total amount of score the player has gained on ranked or approved maps, regardless of finishing the beatmap or setting a new score. In short, relative experience of the player in the game. However, unlike other games (read: MMO games) you might have played, osu! does **NOT** give any advantages to any players whatsoever. A naturally gifted level 10 player can potentially beat a level 100 player (this means that all the level says is that the level 100 player spent more time playing beatmaps online). Also, keep in mind that you get higher scores when you get higher combos (hitting notes without missing any in a row).
-
-### Performance Points
-*Full explanation: [Performance Points][Performance_Points wikilink]*
-
-Performance Points (pp) is a ranking metric aiming to be more contextually relevant to a continuous game like osu!. It aims to shift the focus from being on the amount of time played to the actual skill of the player. Performance points are heavily based on map difficulty which is determined by a unique algorithm in every gamemode. Based on this difficulty alone each of your scores is rated and assigned a pp value. Your performance rank is predominantly based on your performance on individual maps. The easiest way to improve it is to improve your scores on difficult songs. Do note that game modes are counted separately \[which means, playing (osu!/osu!taiko/osu!catch/osu!mania) raises (osu!/osu!taiko/osu!catch/osu!mania) pp\].
-
-If you have any questions regarding pp, please refer to the [wiki page for pp first][Performance_Points#FAQ wikilink].
-
-### Accuracy
-_Full explanation: [Accuracy][Accuracy wikilink]_
 
-![A screenshot depicting a player&#39;s accuracy after completing a beatmap.][Accuracy results image]
+Welcome to _osu!_, a free-to-win rhythm game—developed by peppy—with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drum drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
 
-This measurement is the average accuracy among all the ranked scores the player played. It is split into two types, player's overall and pp's accuracy. In your player tab in-game, the accuracy shown is pp's accuracy. For player's overall, you may have to visit the "Online Results" after you played a beatmap online or refer to your own forum page. In osu!, accuracy measures not only your aim, but the timing as well, so it is important to tap the notes perfectly to the beat. Pay extra attention to the approach circles!
+## Current events
 
-### Achievements
-_Full explanation: [Achievements][Achievements wikilink]_
+<!--There are currently no active events within the osu! community at this time.-->
```

Comment:
"currently" and "at this time" mean the same thing, so one of them is redundant. (i know this is commented out, but it will spare you this comment in the future)